### ISSUE TITLE
(EE) could not connect to wayland server

### ISSUE THREAD

**Issue by allmazz** [AsahiLinux/muvm#108]
When I'm trying to run anything via muvm, it stucks without any messages.
If I press Ctrl-C, following message occurs: "(EE) could not connect to wayland server", so I enabled debug logs and would be glad if anyone could help me, because I do not see any critical issues here.

muvm: 0.1.3-1.fc40
sommelier: 0~20240902git67bc17e-4.fc40
Asahi Fedora Remix with Gnome, all packages up-to-date.

Thanks!
[muvm.log](https://github.com/user-attachments/files/17805197/muvm.log)

**Comment by slp** [comment]
Looks like it can't connect to your compositor. Is there any chance that the `WAYLAND_DISPLAY` environment variable is not exported in the session you're running muvm or it's value is somehow wrong?

**Comment by allmazz** [comment]
no, I can run graphical applications from terminal without any issues. 
WAYLAND_DISPLAY=wayland-0

**Comment by allmazz** [comment]
@asahilina @slp could you please take a look?

**Comment by asahilina** [comment]
As of a recent virglrenderer update it should fail to initialize graphics if the version is mismatched with the kernel (and mesa should also fail to initialize if it is mismatched with virglrenderer).

**Comment by allmazz** [comment]
thank you for reply! I will check virglrenderer and mesa versions

**Comment by allmazz** [comment]
I've downloaded all available updates and reinstalled mesa packages(`dnf reinstall *mesa*`), now it works, thanks.
